src/App.js

```diff
@@ -1,5 +1,11 @@
+import FoodList from './FoodList'
+
 function App() {
-  return <div className="App">hello</div>;
+  return (
+    <div>
```

Comment:
You could use <> & </> instead of divs.

Reply:
But these are more to bundle html elements, if you have more container on the same level. Because react can only return one element.